src/dbos.ts

```diff
@@ -1244,23 +1244,16 @@ export class DBOS {
   /**
    * Start a workflow in the background, returning a handle that can be used to check status,
    *   await the result, or otherwise interact with the workflow.
-   * @param func - The function to start.  If a class or instance method, supply `this` within `params`.
-   * @param params - `StartWorkflowFunctionParams` which may specify the ID, queue, `this`, or other parameters
-   * @param args - Arguments passed to `func`
+   * The full syntax is:
+   * `handle = await DBOS.startWorkflow(<target function>, <params>)(<args>);`
+   * @param func - The function to start.
+   * @param params - `StartWorkflowParams` which may specify the ID, queue, or other parameters for starting the workflow
    * @returns `WorkflowHandle` which can be used to interact with the started workflow
    */
-  static async startWorkflowFunction<This, Args extends unknown[], Return>(
-    params: StartWorkflowFunctionParams<This> | undefined,
-    func: (this: This, ...args: Args) => Promise<Return>,
-    ...args: Args
-  ): Promise<WorkflowHandle<Return>> {
-    const regOp = getRegistrationForFunction(func);
-    if (!regOp) {
-      throw new DBOSNotRegisteredError(func.name, `${func.name} is not a registered DBOS workflow function`);
-    }
-    return await DBOS.#invokeWorkflow(params?.instance, regOp, args, params ?? {});
-  }
-
+  static startWorkflow<Args extends unknown[], Return>(
```

Comment:
How does this JSDoc work in VSCode -- does it automatically pick the correct one based on the input parameter type? Should we use the `@overload` tag in JSDoc? https://devblogs.microsoft.com/typescript/announcing-typescript-5-0-beta/#overload-support-in-jsdoc

Reply:
I don't think we need `@overload`. `startWorkflow` already had two overloads and they didn't have the jsdoc overload tag. But I will research more